json-test.java

```diff
@@ -1,246 +0,0 @@
-package com.vaticle.typedb.example.genealogy.math_genealogy;
-
-import com.vaticle.typeql.lang.query.TypeQLDelete;
-import mjson.Json;
-
-import java.io.IOException;
-import java.nio.file.Files;
-import java.util.*;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-
-import static mjson.Json.read;
-import com.vaticle.typedb.client.api.TypeDBClient;
-import com.vaticle.typedb.client.api.TypeDBSession;
-import com.vaticle.typedb.client.api.TypeDBTransaction;
-import com.vaticle.typedb.client.TypeDB;
-import com.vaticle.typeql.lang.TypeQL;
-
-import static com.vaticle.typeql.lang.TypeQL.*;
-
-import com.vaticle.typeql.lang.query.TypeQLInsert;
-
-class json_test
-{
-    public static int get_id(Json file, int idx)
-    {
-        return file.at("nodes").at(idx).at("id").asInteger();
-    }
-    public static String get_name(Json file, int idx)
-    {
-        return file.at("nodes").at(idx).at("name").asString();
-    }
-    public static String get_school(Json file, int idx)
-    {
-        return  file.at("nodes").at(idx).at("school").asString();
-    }
-    public static int get_year(Json file, int idx)
-    {
-        if (!file.at("nodes").at(idx).at("year").isNull())
-        return file.at("nodes").at(idx).at("year").asInteger();
-        else return 0;
-    }
-    public static String get_speciality(Json file, int idx) {
-        if (!file.at("nodes").at(idx).at("subject").isNull())
-        {
-            return file.at("nodes").at(idx).at("subject").asString();
-        }
-        else return "Unknown";
-    }
-    public static List get_advisors(Json file, int idx) {
-        List<Object> ls = file.at("nodes").at(idx).at("advisors").asList();
-        return ls;
-    }
-
-
-    public static void main(String[] args)
-    {
-        Path filePath=Paths.get("genealogy/math_genealogy/datalite.json");
-
-        TypeDBClient client = TypeDB.coreClient("localhost:1729");
-        try
-        {
-            String contents;
-            contents=Files.readString(filePath);
-            Json file=read(contents);
-            List mylist=file.at("nodes").asList();
-            int sz=mylist.size();
-            System.out.println("Size = "+sz);
-
-            try (TypeDBSession session = client.session("onboarding", TypeDBSession.Type.DATA)) {
-
-                //first purge existing data
-                try (TypeDBTransaction writeTransaction = session.transaction(TypeDBTransaction.Type.WRITE))
-                {
-                    TypeQLDelete query = TypeQL.match(
-                            var("p").isa("thing")
-                    ).delete(var("p").isa("thing"));
-                    writeTransaction.query().delete(query);
-                    // to persist changes, a write-transaction must always be committed (closed)
-                    writeTransaction.commit();
-                }
-
-                //for instantiating mathematicians
```

Comment:
Can we move each of these sections to a function so the main isn't this long to navigate?
You should be able to do it and just pass the session and file to each function.

e.g. `instantiateMathematicians(...)`

Reply:
done